.ci.yaml

```diff
@@ -1062,12 +1062,10 @@ targets:
         }
 
   ### iOS+macOS tasks ###
-  # TODO(stuartmorgan): Move this to ARM once google_maps_flutter has ARM
-  # support. `pod lint` makes a synthetic target that doesn't respect the
-  # pod's arch exclusions, so fails to build.
-  - name: Mac_x64 check_podspecs
+  - name: Mac_arm64 check_podspecs
```

Comment:
Per discussion in the other issue, you could rename this `repo_checks` now to save another 2-PR cycle later.

Reply:
Good idea.